### ISSUE TITLE
Paypal missing image?

### ISSUE THREAD

**Issue by LBManiac** [gburton/Responsive-osCommerce#547]
This seems to happen quite frequently with the paypal image in checkout showing as missing link.

The image is meant to be https://www.paypalobjects.com/webstatic/mktg/Logo/pp-logo-100px.png but when trying to visit this directly it just shows a blank page and in turn creates a missing link icon.

I resolved this by changing the paypal_standard.php image link to one which seems to work without issue https://www.paypalobjects.com/webstatic/mktg/Logo/pp-logo-150px.png

Not sure if there is something wrong with Paypal but it seems strange a main branding image would not be online

**Comment by gburton** [comment]
@LBManiac 

Referenced three times:
https://github.com/gburton/Responsive-osCommerce/search?utf8=%E2%9C%93&q=pp-logo-100px.png&type=

https://www.paypalobjects.com/webstatic/mktg/Logo/pp-logo-100px.png seems to load OK for me?

**Comment by LBManiac** [comment]
Yes it seems to be working at the moment.  Not sure what is causing the image to not be available at times, it is officially listed on the paypal branding page but several times in the past few weeks the image has not loaded in.  I have checked it directly in incognito and phone and each time it returns blank or missing image icon even on the paypal page showing their branding collateral.

If anything it is a PayPal issue, I decided to switch to the 150px image as it seems more reliable but thought it may be worth mentioning as I noticed the broken image several times in a couple of weeks when testing the checkout procedure.
